xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -71,6 +78,9 @@ window.MSCheckbox = Class.create({
     //add new image
     var img = document.createElement('img');
     img.src = this.images[state];
+    img.alt = this.domNode.getAttribute('data-title') + " " + this.alts[state];
```

Comment:
Combining several translation messages in a fixed order is a bad practice and incompatible with some languages. If you need to combine messages, one of them should have a placeholder for the other such that the translator can determine its position. See also https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HRulesfortranslatingcompositeandlongmessages